enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -753,6 +776,9 @@ func (p *PebbleCache) maxDatabaseVersion() filestore.PebbleKeyVersion {
 }
 
 func (p *PebbleCache) activeDatabaseVersion() filestore.PebbleKeyVersion {
+	if p.activeKeyVersion < 0 {
```

Comment:
Should we just set activeKeyVersion = maxDBVersion at the same point where you initialize maxDBVersion above? I could see this causing a confusing bug if this method returns maxDBVersion but the field itself contains a different value (-1)

Reply:
Yeah, I went back and forth about which approach to use. Updated to set the active version after cache creation. Let me know what you think of this approach.

Reply:
Did you mean to remove this check here (p.activeKeyVersion < 0). I believe now p.activeKeyVersion can never be -1?

Reply:
Oh yeah, missed this one. Fixed.